PsfShimMonitor/PsfShimMonitor.csproj

```diff
@@ -7,8 +7,8 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{BC76E43C-2E55-4BFD-95DF-DA09884F6DC4}</ProjectGuid>
```

Comment:
When opening the PsfMonitor project I can see a lot of references, for example: Dia2Lib, TraceReloggerLib, OSExtensions... are they all required?